File: Data Structures & Algorithms/two-integer-sum/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        let obj = new Map();

        for(let i = 0; i < nums.length; i++){
            let diff = target - nums[i]

            if(obj.has(diff)){
                return [obj.get(diff), i]
            }

            obj.set(nums[i], i)

        }
        return []
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        obj = {}

        for num in nums:
            if num in obj:
                obj[num] += 1
            else:
                obj[num] = 1
        
        new_obj = sorted(obj, key=obj.get)

        return new_obj[-k:]

        